.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: pipenv-setup
+  name: pipenv-setup
+  description: "`pipenv-setup` is a command-line utility to sync pipfile/lockfile to setup.py "
+  entry: pipenv-setup sync
```

Comment:
Would it make sense to use the `args` property to provide the `sync` command? This way, developers would be able to override it for calling e.g. `check` instead.

I'm also wondering if we should include the `--dev` argument by default? I've personally run into issues where incompatibilities arise between minimal dependencies and `dev` dependencies when I was excluding that argument

Reply:
Ah yeah that works too, I wanted to limit the number of arguments and have sane default, but that's definitely possible.

For pre-commit, if the tool knows what needs to run I'd rather have the tool do the work directly more than telling me I have to do the work (ie, run the specific command) ;-)

I don't want to expand the scope of this PR but I wanted to make `--pipfile` the default, without `--dev` as it seems more natural flow : 
don't lock the package too much in setup.py otherwise you won't be able to install it (packageA wants requests==2.27.1 ; packageB wants requests==2.27.0 --> no installation possible)

Reply:
> For pre-commit, if the tool knows what needs to run I'd rather have the tool do the work directly more than telling me I have to do the work (ie, run the specific command) ;-)

Agreed 😊

> I wanted to make `--pipfile` the default

Yes please do! I don't think this would introduce too much scope creep.